docs/conf.py

```diff
@@ -60,11 +62,15 @@ def setup(app):
     'sphinxarg.ext',
     'matplotlib.sphinxext.plot_directive',
     'sphinxcontrib.napoleon',
-    'm2r'
+    'm2r',
+    'nbsphinx'
 ]
 
 napoleon_google_docstring = False
 
+# Remove this once decode_trf docs are live
+nbsphinx_allow_errors = True
```

Comment:
@Centrus007 to ease reviewing of the new docs I have temporarily allowed docs to build even if the examples error. As you will see, without this the following example fails miserably and readthedocs fails to build:

https://pymedphys.com/en/unreleased-docs/examples/trf_mudensity.html

Reply:
That did make life easier, thank you :)